File: 프로그래머스/0/120860. 직사각형 넓이 구하기/직사각형 넓이 구하기.js
function solution(dots) {
    const x_dots = dots.map(dot => dot[0]);
    const y_dots = dots.map(dot => dot[1]);
    
    const x = Math.max(...x_dots) - Math.min(...x_dots);
    const y = Math.max(...y_dots) - Math.min(...y_dots);
    
    return x * y;
}

File: 프로그래머스/0/120860. 직사각형 넓이 구하기/직사각형 넓이 구하기.py
def solution(dots):
    x_dots = [dot[0] for dot in dots]
    y_dots = [dot[1] for dot in dots]
    
    x = max(x_dots) - min(x_dots)
    y = max(y_dots) - min(y_dots)
    
    return x * y